uptycs/README.md

```diff
@@ -0,0 +1,85 @@
+# Uptycs
+
+## Overview
+
+Your developer's laptop is just a hop away from cloud infrastructure. Attackers don't think in isolated systems, so why would you rely on fragmented solutions to protect public cloud, private cloud, containers, laptops, and servers? Uptycs mitigates risk by prioritizing your responses to threats, vulnerabilities, misconfigurations, sensitive data exposure, and compliance requirements across your modern attack surface—all accessible through a single user interface and data model. This includes the capability to correlate threat activity as it traverses on-prem and cloud boundaries, providing a more comprehensive enterprise-wide security posture. Looking for acronym coverage? We've got you covered with CNAPP, CWPP, CSPM, KSPM, CIEM, CDR, and XDR. Start with your Detection Cloud, utilize Google-like search, and the attack surface coverage you need today. Stay prepared for what lies ahead.
+
+For more, please visit: [Uptycs][1]
+
+The [Uptycs][1] integration enables you to ingest your Uptycs alerts and detection into Datadog events. 
+## Setup
+
+To set up this integration, please [Contact Us][2] for an Uptycs account.
+You'll also need Datadog API keys.
+### Configuration
```

Comment:
@anawishnoff As discussed previously, we checked for feasibility of OAuth. But each customer gets different domain with Uptycs platform. So currently it is not possible. Hence we updated our UI so that users can easily configure DataDog destination for Uptycs alerts and events. Updated the doc and screenshots. Please check.